diff --git a/arch/arm64/boot/dts/renesas/imdt-v2h-sbc.dts b/arch/arm64/boot/dts/renesas/imdt-v2h-sbc.dts
index d109571455d2..9f3527f0a13d 100644
--- a/arch/arm64/boot/dts/renesas/imdt-v2h-sbc.dts
+++ b/arch/arm64/boot/dts/renesas/imdt-v2h-sbc.dts
@@ -119,6 +119,12 @@ &usb2_phy1 {
 	status = "okay";
 };
 
+&drp1 {
+	memory-region = <&drp_codec>;
+	memory-oca-region = <&opencva_reserved>;
+	status = "okay";
+};
+
 &adc {
 	status = "okay";
 };
